roles_n_rules/switch.py

```diff
@@ -5,16 +5,17 @@
 from roles_n_rules.functions import cupid_kiss
 import random
 import roles_n_rules.role_data as roles
+import story_time.powerup as power
 import config
 
 def pay():
     """This function takes care of all properties that need to happen in the first wave of the end of the night.
     The function returns a Mailbox."""
```

Comment:
Usually you would go with a
```py
def pay() -> Mailbox:
```
but it is okay

Reply:
Whaddaya mean?

Reply:
It literally means: this function returns a `Mailbox`